cluster-autoscaler/cloudprovider/azure/test/templates/cluster-template-prow-aks-aso-cluster-autoscaler.yaml

```diff
@@ -162,11 +162,6 @@ spec:
       mode: System
       owner:
         name: ${CLUSTER_NAME}
-      tags:
```

Comment:
Should we try to isolate things like kube-system Pods from the side effects of these tests? That's what I was trying to achieve here by removing the autoscaler tags from this System pool but I can still see things like coredns sometimes scheduling onto User pools.

Reply:
I'm not sure about the connection here, but I think at this phase of the game this is good enough, and we're in good shape to raply make tweaks to cover these types of things that make testing cluster-autoscaler tricky.